xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -111,6 +111,21 @@ void CSelectionStreams::Clear(StreamType type, StreamSource source)
   }
 }
 
+void CSelectionStreams::Clear(StreamType type, int source)
```

Comment:
we have this already
CSelectionStreams::Clear(StreamType type, StreamSource source)
you added
CSelectionStreams::Clear(StreamType type, int source)

makes no sense to me